### ISSUE TITLE
Deprecate stickies

### ISSUE THREAD

**Issue by markkap** [calmPress/calmpress#102]
It is a useless feature for people that do not run a blog, and probably too hidden for those that do. At this point it is just bloat.

**Comment by mikeschinkel** [comment]
We have used this feature for some of our clients sites that are news organizations, so it does have more use than just blogs.

**Comment by markkap** [comment]
@mikeschinkel exactly the real life feedback I was looking for ;). I do think that that there are systematic problems with the definition and implementation of stickies, the way they are only available on posts and not other CPTs and the surprising way you are going to get them in query results if you are not careful. 
But maybe there should be an alternative solution first

**Comment by mikeschinkel** [comment]
@markkap Yes, I would by no means assume that the current implementation is ideal. That said, I have never spent much time pondering this particular concern so I have no particular suggestion for improvements at the moment.
